docs/style.css

```diff
@@ -0,0 +1,91 @@
+body {
+  overflow-x: hidden;
+}
+.feature-title {
+  text-align: center;
+}
+.left-align {
+  text-align: left;
+}
+.right-align {
+  text-align: right;
+}
+.bg-color-orange {
+  background: rgb(237, 89, 47) !important;
+}
+.browser-info {
+  text-transform: capitalize;
+}
+.feature-content {
+  padding-left: 6em !important;
+  padding-right: 6em !important;
+}
+
+.responsive-img {
+  height: auto;
+  width: 100%;
+  max-width: 500px;
+  min-width: 150px;
+}
+.grid {
+  padding-left: 4em;
+  padding-right: 4em;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
+  justify-items: start;
+  background: rgb(245, 245, 245);
+}
+.grid > div {
+  padding: 2em;
+  margin: 0.5em;
+}
+.grid2 {
+  padding-left: 2em;
+  padding-right: 2em;
+  display: grid;
+  grid-template-columns: 2fr 3fr;
+  grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
+  justify-items: end;
+}
+.grid2 > div {
+  padding: 2em;
+  margin: 0.5em;
+}
+
+.button {
+  font-size: large;
+}
```

Comment:
You can remove this class. This is overriding the vocabulary defaults and in this case the default size looks good.

**Before**

![Screenshot from 2020-03-31 21-44-10 (1)](https://user-images.githubusercontent.com/34679965/78050481-e78a1c00-7399-11ea-8e9f-e63a7f61dbbd.png)

**After**

![Screenshot from 2020-03-31 21-44-27](https://user-images.githubusercontent.com/34679965/78050284-a98cf800-7399-11ea-8c9a-06a3ae083e58.png)